otj-server-mvc/src/main/java/com/opentable/server/mvc/ExceptionLogFilter.java

```diff
@@ -0,0 +1,68 @@
+package com.opentable.server.mvc;
+
+import java.io.IOException;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+
+import org.eclipse.jetty.io.QuietException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+import org.springframework.stereotype.Component;
+import org.springframework.web.util.NestedServletException;
+
+
+@Component
+@Conditional(ExceptionLogFilter.InstallExceptionLogFilter.class)
```

Comment:
A couple of comments, which I'd appreciate an answer @dkaukov 
1. Don't understand why this is MVC only.
2. Don't at all understand or agree why we dont simply rethrow the original exception? The whole point unless I'm missing something is simply to catch, log, and rethrow.

Reply:
the idea was to log message in context of filter chain, so it would have MDC with request-id. When it is bubled-up to Jetty, it looses MDC. but also did not wanted to log same thing multiple times, so I'm throwing SilentException.

Have no idea, is it needed for jax-rs.. ;(

Reply:
Moved to core. jax-rs had the same problem